README.md

```diff
@@ -2,7 +2,17 @@
 
 Backend for Resource Optimization Service
 
+The Red Hat Insights resource optimization service enables RHEL customers to assess and monitor their public cloud usage and optimization. The service exposes workload metrics for CPU, memory, and disk-usage and compares them to resource limits recommended by the public cloud provider.
+Currently ROS only provides suggestions for AWS RHEL instances. To enable ROS, a customer needs to perform a few prerequisite steps on targeted systems via Ansible playbook.
 
+Underneath, ROS uses Performance Co-Pilot (PCP)  to monitor and report workload metrics.
+
+## How it works
+
+![UML](https://www.plantuml.com/plantuml/png/fLDDR-f65DtxLrnD5zs0-rwxGn5w8Q5Lf2GW72Ab0Lc3vmAZs3FMV92WrF-zTx0TaJgmw6E1FEUU-ttk3-kuSRx88oTTZZ2R933YsGig0LEZCxHMgWtycUlNg2Icp6J9bOFjIaQeo6SFPSxTMfiYbcf7a3Rx6CE7_UJ0fB9oitLseL1jf4BMyQ_nsA3rENclz1kmeteO_78R8enoZn7GxtFuDZdKoHUghClNXdNPTIgeKttcXiufTIIuuojkiQxXUiGYwha_FDdj0BRQEagpppKNqKbUV_Z5ABfGawaxbm6qt7wcpKSFqGawfJlJxBrsP0ZHGA_XPCPtgg_qZWtWuhZ9OM-XcDkNI0RcIveO2bOH-CuQLnPyxTuwB0AdGCV3N0CSnrLBBRKvK1VgcPu-nrGAUlvgj4sfMSTLXYdkUEwvarhzSv4y9G_ZkpHvk7mO3wibLI4eiin3a13omwajplqU5p8pceMA3MueIe2zxIEthl_zW9EFGmeH8cIb9y8pkvhEqydZU9RS_3s-PZrWSSoMWLnWGJizSEx6TvFPqzSqgGFbPfBExwyRSpfwIgUpSPBCP-DbhwgMLHMTsT55lnNW0Fv21ppBj5SEcevXRNG1istQu1P8jZb6ZHEvduazW2k3r3Jy1lF3-g3U5KYrrlG7oU7vap5BDEkK26kP8v2-e97AGdraWocQeBomj6dDh7F6Pm-fU_JdjTqDKwkxDQQjG1iTDJCDkc_7hdonGZFiaPrvamVJV7OBJeFGhofS1N2JRUKEwyqaVzHSQLEZWpwTyHY4py8bDNdgERGpQi0irru4R2S5cY1N_cf9Myt1r53fIwTrGAK8jKWnh8wlCZDuk4h2_djz9BtZZNPd-Nzrib2NGX3r4OrSxrl1qaq3LmAqsT1dSEgArhv8LNB32nZfjoaNGgeDVF_sBVg1IlWY_nS0)
```

Comment:
In this diagram, at engine result processor component, you mentioned `Create / Update perf_profile for updated\nsystem in PerformanceProfile table`.
Though method name says create_or_update, every time it creates fresh performance profile record.
could you modify the statement something like - `Create Performance Profile record for a system in  table`?